packages/client/hmi-client/src/types/Project.ts

```diff
@@ -3,4 +3,11 @@ export type Project = {
 	name: string;
 	description: string;
 	timestamp: string;
+	assets: {
+		publications: number[];
+		intermediates: number[];
+		models: number[];
+		plans: number[];
+		simulation_runs: number[];
+	};
```

Comment:
Same comment I gave @amostafa-uncharted: Why not putting those assets on the root? they are as important part of a project than `name` or `timestamp`?
Also, we don't care about respecting the _Data Service_ types on the front-end.